config/routes.rb

```diff
@@ -27,6 +27,11 @@
                  constraints: ->(_) { FeatureFlipper.show_education_benefit_form? }
     end
 
+    resource :health_care_application, only: [:create, :index] do
+      post :create
+      get :index
```

Comment:
If you declare it as a `resource`, you are saying the user can only access a single health care application, so `index` would not be used to show all the applications, and `show` would be action responsible for the GET request to `/v0/health_care_application`. Will a veteran have multiple health care applications, or just one? What operations can they take, just creation, or can they view their application as well?
If you change it to `resources`, the name should become plural
`resources :health_care_applications`